hc/api/tests/test_create_check.py

```diff
@@ -16,6 +17,7 @@ def post(self, data, expected_error=None):
 
         if expected_error:
             self.assertEqual(r.status_code, 400)
+            self.assertIn(expected_error, r.content.decode())
             ### Assert that the expected error is the response error
```

Comment:
where is the test for this?

Reply:
I used the assertIn to check if the response contained text equalling the error text message